include/swift/SIL/GraphOperationInfo.h

```diff
@@ -69,11 +69,18 @@ struct GraphOperationInfo {
     UnknownShapeListAttribute,
 
     /// A metatype of a TensorFlow value type or aggregate of TensorFlow value
-    /// types that should be lowered into a list of type attributes.
+    /// types. Deabstraction lowers this to TFDataTypeAttribute.
     ///
     /// Written as named argument with "$typeList" suffix.
     TypeListAttribute,
```

Comment:
from PR description, it seems we keep this enum value only for short-term concerns. should we add a TODO to remove it?

Reply:
yep, will add TODO